core/src/main/scala/org/ensime/core/javac/JavaCompletion.scala

```diff
@@ -208,21 +208,26 @@ trait JavaCompletion extends Helpers with SLF4JLogging {
     importing: Boolean,
     caseSense: Boolean
   ): List[CompletionInfo] = {
-    val candidates = typeElement(info, target).map { el =>
-      el match {
-        case tel: TypeElement => {
-          val elements: Elements = info.getElements()
-          elements.getAllMembers(tel).flatMap { e =>
-            filterElement(info, e, prefix, caseSense, importing, false)
-          }
+
+    typeElement(info, target).map {
```

Comment:
is this maybe better as a `flatMap`? I realise this is just code you're changing but no reason not to clean it up at the same time.